samples/explore_datasource.py

```diff
@@ -9,56 +9,61 @@
 # on top of the general operations.
 ####
 
-
-import tableauserverclient as TSC
-import os.path
 import argparse
 import getpass
 import logging
 
+import tableauserverclient as TSC
+
+
 parser = argparse.ArgumentParser(description='Explore datasource functions supported by the Server API.')
```

Comment:
I debated that too -- I wonder if I should wrap everything in a main(), and then just have:

```
if __name__ == "__main__":
    main()
```

So it's easier to tell runtime from any setup?